spec/support/serializers/organisation_serializer.rb

```diff
@@ -3,6 +3,7 @@ class OrganisationSerializer < JSONAPI::Serializable::Resource
   has_many :organisation_users
 
   has_many :users, through: :organisation_users
+  has_many :providers
```

Comment:
The requirement for this is not-so-obvious. One idea for making it more obvious is to add a validation to `stub_api_v2_resource`, `stub_api_v2_resource_collection`, etc, that would check that any included resources show up in the generated `jsonapi_response`, and raise an error if it's missing. This would imply that it's missing from the serialiser here.

Reply:
I think this is going to be harder than originally thought because of the through includes i.e. provider include `courses.accrediting_provider`.